docs/tutorials/creating-your-first-substrate-chain/learn.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Background Information
+---
+
+In this section we will teach you about the Substrate blockchain development
+framework.
+
+## Background on Blockchains
+
+Blockchain development is hard.
+
+Blockchain networks are composed of individual **nodes** that are connected
+together on a peer-to-peer (P2P) network. Nodes are the individual computers on
+a network running the blockchain software that makes everything work.
+
+To function, a blockchain node needs:
+
+* A Database
```

Comment:
why are all the title cased? these are all normal terms, none of them a  special name...